### PR TITLE
Feature project create view

### LINKED ISSUE #50: Manager Base Interface
- [ ]  Dashboard with available projects
- [ ]  Interface with people assigned to projects (feature moved to creating raport feature)
- [ ]  Interface with all details of chosen project

Manager Base Interface is directed to the Admin-type and Manager-type user
- for Admin-type user:
- [x] **List** with available projects
- [x] Interface with all **details** of chosen project
- [x] Interface for **updating** details of chosen project
- [ ] Interface for **creating** new project
- [ ] Interface for **deleting** project

- for Manager-type user:
- [x] **List** with manager's projects
- [x] Interface with all **details** of manager's project
- [x] Interface for **updating** details of manager's project
- [ ] Interface for **deleting** manager's project

### LINKED ISSUE #51: Create tests for projects views
Create necessary tests for managers app views

- [ ] Tests for ProjectsList APIviews

- [ ] Tests for ProjectsDetail APIviews

- [ ] Tests for ProjectsUpdateAPIviews

- [ ] Tests for ProjectsCreate APIviews

- [ ] Tests for ProjectsDelete APIviews

### PR DESCRIPTION
**Pull request by MartynaAnnaGottschling**
Resolves #50
Resolves #51

### CODE REVIEW

File: managers/templates/managers/project_create.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <link rel="stylesheet" type="text/css" href="{% static 'managers/tooltip.css' %}" />
+</head>
+
+<h2>
+  <small>
+    <a href="{% url 'custom-projects-list' %}">
+      <span class="glyphicon glyphicon-chevron-left"></span>
+    </a>
+  </small>
+  &nbsp;Create new project&nbsp;
```
Comment (kbeker) [inline managers/templates/managers/project_create.html]: Please add translation here

File: managers/templates/managers/project_create.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+</h2>
+
+<div class="modal-dialog" style="margin-bottom:0">
+  <form action="{% url 'custom-project-create' %}" method="post">
+      {% csrf_token %}
+      {% render_form serializer template_pack='rest_framework/vertical' %}
+          <div class="tool_tip">
+            <font color="grey">How to add more employees?</font>
```
Comment (kbeker) [inline managers/templates/managers/project_create.html]: And everywhere where it is needed

File: managers/tests/test_unit_managers_apiview.py [hunk 3]
```diff
@@ -38,6 +38,7 @@ def setUp(self):
             reverse('custom-projects-list'),
             reverse('custom-project-detail', args=(self.project.pk,)),
             reverse('custom-project-update', args=(self.project.pk,)),
+            reverse('custom-project-create'),
```
Comment (kbeker) [inline managers/tests/test_unit_managers_apiview.py]: Same as: https://github.com/Code-Poets/sheetstorm/pull/85#discussion_r264261234

File: managers/tests/test_unit_managers_apiview.py [hunk 4]
```diff
@@ -127,9 +128,52 @@ def test_project_list_view_should_display_projects_sorted_by_members_count_desce
...
+        self.assertEqual(response.status_code, 200)
+        self.assertNotEqual(response.data.get('errors'), None)
+        self.assertTrue('name' in response.data.get('errors'))
+        self.assertEqual(Project.objects.filter(name='Another Example Project').count(), 0)
+
+
 class ProjectDetailTests(ProjectTest):
     def test_project_detail_view_should_display_project_details_on_get(self):
-        request = APIRequestFactory().get(path=self.url[1])
+        request = APIRequestFactory().get(path=self.url[2])
```
Comment (kbeker) [inline managers/tests/test_unit_managers_apiview.py]: Why it is changed?

File: managers/templates/managers/project_create.html (hunk 1 above)
Comment (MartynaAnnaGottschling) [reply managers/templates/managers/project_create.html]: Done

File: managers/templates/managers/project_create.html (hunk 2 above)
Comment (MartynaAnnaGottschling) [reply managers/templates/managers/project_create.html]: Done

File: managers/tests/test_unit_managers_apiview.py (hunk 4 above)
Comment (MartynaAnnaGottschling) [reply managers/tests/test_unit_managers_apiview.py]: Removed

File: managers/tests/test_unit_managers_apiview.py (hunk 3 above)
Comment (MartynaAnnaGottschling) [reply managers/tests/test_unit_managers_apiview.py]: Done

File: managers/views.py [hunk 5]
```diff
@@ -54,10 +58,32 @@ def get(_request: HttpRequest, pk: int) -> Response:
...
+    template_name = "managers/project_form.html"
+
+    def get_context_data(self, **kwargs: Any) -> dict:
+        context_data = super().get_context_data(**kwargs)
+        context_data["title"] = self.object.name
+        context_data["back_url"] = self.get_success_url()
+        return context_data
 
-    def get_success_url(self) -> None:
+    def get_success_url(self) -> str:
```
Comment (kbeker) [inline managers/views.py]: Good catch @rwrzesien :+1:

**Pull request merged**